Unable to obtain body of a post request.

So I have a simple function that makes a simple post request with some parameters:

```python
    async with aiohttp.ClientSession(raise_for_status=True) as session:
        async with session.post(trigger.url, data=params) as response:
            return await response.json()
```

When I try to test the parameters passed:

```python
        aresponses.add("test.url.com", "/trigger/path", "POST", aresponses.Response(body=json.dumps({"some": "response"}), status=201, headers={"Content-Type": "application/json"}))
        response = await utils.invoke_trigger(trigger, variables)
        print(await aresponses.history[0].request.post())
```

I get following error:
```python
self = <StreamReader 75 bytes eof e=PayloadAccessError()>

    async def readany(self) -> bytes:
        if self._exception is not None:
>           raise self._exception
E           aiohttp.web_protocol.PayloadAccessError

.venv/lib/python3.8/site-packages/aiohttp/streams.py:386: PayloadAccessError
```

--- a/aresponses/main.py
+++ b/aresponses/main.py
@@ -179,7 +179,7 @@ class ResponsesMockServer(BaseTestServer):
             return route, response
 
         self._unmatched_requests.append(request)
-        return route, None
+        return None, None
 
     async def passthrough(self, request):
         """Make non-mocked network request"""

--- a/aresponses/main.py
+++ b/aresponses/main.py
@@ -109,6 +109,9 @@ class ResponsesMockServer(BaseTestServer):
     async def _handler(self, request):
         self._request_count += 1
         route, response = await self._find_response(request)
+        # ensures the request content is loaded even if the handler didn't need it. This makes it available in the
+        # `aresponses.history`
+        await request.text()
         self._history.append(RoutingLog(request, route, response))
         return response
 

--- a/tests/test_server.py
+++ b/tests/test_server.py
@@ -309,3 +309,21 @@ async def test_history(aresponses):
     assert aresponses.history[1].request.host == "bar.com"
     assert "Route(" in repr(aresponses.history[0].route)
     aresponses.assert_plan_strictly_followed()
+
+
+@pytest.mark.asyncio
+async def test_history_post(aresponses):
+    """Ensure the request contets exist in the history"""
+    aresponses.add(method_pattern="POST", response={"some": "response"})
+
+    async with aiohttp.ClientSession() as session:
+        async with session.post("http://bar.com/zzz", json={"greeting": "hello"}) as response:
+            response_data = await response.json()
+            assert response_data == {"some": "response"}
+
+    assert len(aresponses.history) == 1
+    assert aresponses.history[0].request.host == "bar.com"
+    request_data = await aresponses.history[0].request.json()
+    assert request_data == {"greeting": "hello"}
+    assert "Route(" in repr(aresponses.history[0].route)
+    aresponses.assert_plan_strictly_followed()

